WalletWasabi.Tests/UnitTests/WabiSabi/Backend/CoinVerifierTests.cs

```diff
@@ -188,7 +188,7 @@ public async Task CanFillWhitelistAfterVerificationTestAsync()
 		CoinJoinIdStore coinJoinIdStore = new();
 		CoinVerifierApiClient apiClient = new("token", Network.Main, mockHttpClient.Object);
 		Whitelist whitelist = new(Enumerable.Empty<Innocent>(), string.Empty, new WabiSabiConfig());
-		CoinVerifier coinVerifier = new(coinJoinIdStore, apiClient, whitelist, _wabisabiTestConfig);
+		CoinVerifier coinVerifier = new(coinJoinIdStore, apiClient, whitelist, _wabisabiTestConfig, auditsDirectoryPath: string.Empty);
```

Comment:
```suggestion
		await using CoinVerifier coinVerifier = new(coinJoinIdStore, apiClient, whitelist, _wabisabiTestConfig, auditsDirectoryPath: string.Empty);
```
(and in other places)

Reply:
Assigning `auditsDirectoryPath: string.Empty` is imho not good. The empty string is not a valid directory and it's not obvious why it should work at all.

Why not simply use the other constructor `internal CoinVerifier(CoinJoinIdStore coinJoinIdStore, CoinVerifierApiClient apiClient, WabiSabiConfig wabiSabiConfig, CoinVerifierAuditArchiver? auditArchiver = null)` and pass `auditArchiver = null`...?

Reply:
Yes, it's messy. The `ctors` didn't get updated, but the class changed a lot.
I will improve the situation here.